flake.nix

```diff
@@ -415,16 +432,23 @@
             };
           });
 
-          # Static building isn't really possible on MacOS because the system call ABIs aren't stable.
+          # Static building isn't really possible on MacOS because the system
+          # call ABIs aren't stable. This output shouldn't be used on MacOS.
           nickel-static =
-            if pkgs.stdenv.hostPlatform.isMacOS
-            then nickel-lang-cli
```

Comment:
the if-else is useless because there is already a guard at the output level, so `nickel-static` is not even available as an output on Darwin to begin with.